packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -73,102 +99,220 @@
 		{
 			"#": "23",
 			"label": "O1 No/Nc",
-			"description": "0",
+			"description": "The set value determines the relay contact type for output O (O1). The relay contact type can be normally open (NO) or normally closed (NC).",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "NO",
+					"value": 0
+				},
+				{
+					"label": "NC",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "25",
 			"label": "Set Timer Units to Seconds Or Milliseconds For O1",
-			"description": "0",
+			"description": "Set the timer units to seconds or milliseconds. Choose if you want to set the timer in seconds or milliseconds in Parameters No. 19, 20.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "timer set in seconds",
+					"value": 0
+				},
+				{
+					"label": "timer set in milliseconds (scale 10ms)",
+					"value": 1
+				}
+			]
+		},
+        {
+			"#": "36",
+			"label": "O (O1) Power report on change - percentage",
+			"description": "This parameter determines the minimum change in consumed power that will result in sending a new report to the gateway.",
+			"unit": "%",
+            "valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 50,
+			"unsigned": true
+		},
+        {
+			"#": "39",
+			"label": "Minimum time between reports (O) O1",
+			"description": "This parameter determines the minimum time that must elapse before a new power report on O (O1) is sent to the gateway.",
+			"unit": "seconds",
+            "valueSize": 1,
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 30,
 			"unsigned": true
 		},
 		{
 			"#": "91",
 			"label": "Alarm Conf. - Water",
-			"description": "0",
+			"description": "This parameter determines which alarm frames the Device should respond to and how. The parameters consist of 4 bytes, the three most significant bytes are set according to the official Z-Wave protocol specification.",
 			"valueSize": 4,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "no action",
+					"value": 0
+				},
+				{
+					"label": "open relay",
+					"value": 1
+				},
+				{
+					"label": "close relay",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "92",
 			"label": "Alarm Conf. - Smoke",
```

We try to avoid abbreviating words in labels
```suggestion
			"label": "Alarm Configuration: Smoke",
```